src/tools/error_index_generator/main.rs

```diff
@@ -286,7 +286,7 @@ fn main() {
     let (format, dst) = parse_args();
     let result = syntax::with_default_globals(move || main_with_result(format, &dst));
     if let Err(e) = result {
-        panic!("{}", e.description());
+        panic!("{}", e.to_string());
```

Comment:
Just looking at this from TWIR, but couldn't this leave out the `.to_string()` to avoid double-allocation of the message?

Reply:
If e impl Display, does it turn to

```suggestion
        panic!("{}", e);
```